install/netbox-install.sh

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 community-scripts ORG
+# Author: bvdberg01
+# License: MIT
+# https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+
+source /dev/stdin <<< "$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get update
+$STD apt-get install -y curl
+$STD apt-get install -y sudo
+$STD apt-get install -y mc
+$STD apt-get install -y apache2
+$STD apt-get install -y redis-server
+$STD apt-get install -y postgresql
+$STD apt-get install -y python3
+$STD apt-get install -y python3-pip
+$STD apt-get install -y python3-venv
+$STD apt-get install -y python3-dev
+$STD apt-get install -y build-essential
+$STD apt-get install -y libxml2-dev
+$STD apt-get install -y libxslt1-dev
+$STD apt-get install -y libffi-dev
+$STD apt-get install -y libpq-dev
+$STD apt-get install -y libssl-dev
+$STD apt-get install -y zlib1g-dev
+msg_ok "Installed Dependencies"
+
+msg_info "Setting up PostgreSQL"
+DB_NAME=netbox
+DB_USER=netbox
+DB_PASS=$(openssl rand -base64 18 | tr -dc 'a-zA-Z0-9' | cut -c1-13)
+$STD sudo -u postgres psql -c "CREATE ROLE $DB_USER WITH LOGIN PASSWORD '$DB_PASS';"
+$STD sudo -u postgres psql -c "CREATE DATABASE $DB_NAME WITH OWNER $DB_USER TEMPLATE template0;"
+msg_ok "Set up PostgreSQL"
+
+msg_info "Installing NetBox"        
+RELEASE=$(curl -s https://api.github.com/repos/netbox-community/netbox/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
+wget -q "https://github.com/netbox-community/netbox/archive/refs/tags/v${RELEASE}.tar.gz"
```

Comment:
Why with tarball?
You can do it the same way as with other scripts. Keyword harmonization.

(Not tested, only written)

cd /opt

RELEASE=$(curl -s https://api.github.com/repos/netbox-community/netbox/releases/latest | grep "tag_name" | awk '{print substr($2, 2, length($2)-3) }')

wget -q "https://github.com/netbox-community/netbox/archive/refs/tags/${RELEASE}.zip"
unzip -q ${RELEASE}.zip
mv netbox-${RELEASE:1} /opt/netbox
cd /opt/netbox

Reply:
Because the netbox install guide used tarball, but I have no problems with zip.

Reply:
Converted all tar to zip